src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTag.cs

```diff
@@ -18,18 +32,87 @@ namespace Microsoft.CodeAnalysis.Editor.InlineParameterNameHints
     internal class InlineParameterNameHintsTag : IntraTextAdornmentTag
     {
         public const string TagId = "inline parameter name hints";
+        private readonly IToolTipService _toolTipService;
+        private readonly ITextView _textView;
+        private SnapshotSpan _span;
+        private readonly SymbolKey _key;
+        private readonly IThreadingContext _threadingContext;
+        public Lazy<IStreamingFindUsagesPresenter> _streamingPresenter;
 
         /// <summary>
         /// Creates the UIElement on call
         /// Uses PositionAffinity.Successor because we want the tag to be associated with the following character
         /// </summary>
-        /// <param name="text">The name of the parameter associated with the argument</param>
-        public InlineParameterNameHintsTag(string text, double lineHeight, TextFormattingRunProperties format)
-            : base(CreateElement(text, lineHeight, format), removalCallback: null, PositionAffinity.Successor)
+        /// <param name="adornment">The adornment that we are creating</param>
+        /// <param name="toolTipService">The service being used to create the tooltip and match the UI</param>
+        /// <param name="textView">The view of the editor</param>
+        /// <param name="span">The span that has the location of the hint</param>
+        /// <param name="key">The symbolkey associated with each parameter</param>
+        private InlineParameterNameHintsTag(FrameworkElement adornment,
+                                           IToolTipService toolTipService, ITextView textView,
+                                           SnapshotSpan span, SymbolKey key, IThreadingContext threadingContext,
+                                           Lazy<IStreamingFindUsagesPresenter> streamingPresenter)
+            : base(adornment, removalCallback: null, PositionAffinity.Successor)
         {
+            _streamingPresenter = streamingPresenter;
+            _textView = textView;
+            _key = key;
+            _toolTipService = toolTipService;
+            _span = span;
+            _threadingContext = threadingContext;
+            adornment.ToolTip = "Test";
```

Comment:
Probably choose a different string here, but make a comment it doesn't really matter what we pick.